scripts/release-steps.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Exit on error and output commands.
+set -ex
+
+NEXT_VERSION=$1
+if [ "$NEXT_VERSION" = "" ]; then
+	echo "Error: Version not set!"
+	exit
+fi
+
+# Fix permissions for ssh config and keys.
+chown -R root:root /root/.ssh
+export GIT_SSH_COMMAND="ssh -i /root/.ssh/id_rsa"
+
+# Create working directory and checkout Sensei.
+mkdir /release && cd /release
+git clone git@github.com:Automattic/sensei.git
```

Comment:
In a CI context, we should use the commit hash the pipeline is executing to make sure we are releasing the exact version that we decide to release. 
Maybe we can just assume the clone was done previously.